apps/cow-amm-deployer/src/lib/abis/ConstantProductFactory.ts

```diff
@@ -0,0 +1,276 @@
+export const ConstantProductFactoryABI = [
```

Comment:
This same ABI is used on the composable-cow-api, what you think about creating an ABI package?